### ISSUE TITLE
tf.linalg.sqrtm() returns nan

### ISSUE THREAD

**Issue by hathawayxxh** [tensorflow/tensorflow#31073]
I want to perform the function of matrix square root in my CNN, which can be automatically backpropagated. I used the function tf.linalg.sqrtm() to compute the square root of an invertible matrix. After running the session, it returns an array with all elements equal to "nan". Does anyone know about the reason for the incorrect result? Or any other methods to compute the matrix square root in the forward process and compute gradients automatically during the backward process?
The settings are:
- Linux Ubuntu 16.04
- TensorFlow version: 1. 14. 0
- Python version: 3.6

**Comment by ravikyram** [comment]
@hathawayxxh 
In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
